helmfile/values/grafana-user.yaml.gotmpl

```diff
@@ -1,7 +1,7 @@
 adminPassword: {{ .Values.user.grafanaPassword }}
 
 image:
-  tag: 9.3.13
```

Comment:
Why do we specify this here? As it can not be overridden by the override configs.